BO4E/ENUM/Anfragekategorie.cs

```diff
@@ -0,0 +1,66 @@
+using BO4E.meta;
+
+namespace BO4E.ENUM
+{
+    /// <summary>
+    /// Kategorie der Anfrage (ORDERS ORDRSP BGM 1001)
+    /// </summary>
+    [NonOfficial(NonOfficialCategory.MISSING)]
+    public enum Anfragekategorie
+    {
+        /// <summary>PROZESSDATENBERICHT</summary>
+        /// <remarks>7</remarks>
+        PROZESSDATENBERICHT,
+
+
+        /// <summary>GERAETEUEBERNAHME</summary>
+        /// <remarks>Z10</remarks>
+        GERAETEUEBERNAHME,
+
+        /// <summary>WEITERVERPFLICHTUNG_BETRIEB_MELO</summary>
+        /// <remarks>Z11</remarks>
+        WEITERVERPFLICHTUNG_BETRIEB_MELO,
+
+
+        /// <summary>AENDERUNG_MELO</summary>
+        /// <remarks>Z12</remarks>
+        AENDERUNG_MELO,
+
+        /// <summary>STAMMDATEN_MALO_ODER_MELO</summary>
+        /// <remarks>Z14</remarks>
+        STAMMDATEN_MALO_ODER_MELO,
```

Comment:
![grafik](https://user-images.githubusercontent.com/23094997/155485732-10d90a2d-1c58-4ed8-82db-798cc4790e63.png)

mindestend in FV2210 gibt es da noch mehr

Reply:
Alte Frage, wollen wir jetzt wirklich schon 2210 mit umsetzen?